### ISSUE TITLE
How to save authentification

### ISSUE THREAD

**Issue by bahuma20** [christiansmith/ngGAPI#5]
Is it possible to save the auth-Token in the browser session or localstorage?

**Comment by jcwatson11** [comment]
Yes. The javascript client does this for your automatically. You really don't need to worry about it. If you're curious about how it works, you can search for 'Token' in the source code and you can see how it's used.

**Comment by narengs7** [comment]
@jcwatson11  i get new token each time i reload the page, but i wanted to use single token. Can you suggest me if i am doing any mistake in code.

**Comment by jcwatson11** [comment]
Not without seeing your code. What gives you the idea that you are getting a new token every time you load the page?
